en-US/skins/gameplayinterface.md

```diff
@@ -536,6 +536,23 @@ Notes:
 
 - Background image that displays the current judgements the player has.
 - Automatically colored according to the [JudgeColors skin.ini value]()
+- Can be edited per judgement: `/Judgements/judgement-overlay-{marv, perf, great, good, okay, miss}.png`
```

Comment:
1. Change ``Background image`` to ``Container image``, as saying "background" for an overlay image is confusing.

2. The Overlay is not really ``Automatically colored accordingt o the JudgeColors``, but the animation is, the counter INACTIVE color is half the JudgeColor, and then the ACTIVE color, which is when an hitbobject is hit, is the full JudgeColor (This only apply if ``JudgementCounterFadeToAlpha`` is set to false)

3. Technically this is supposed to be the default behavior now, ``judgement-overlay`` is only there for default/old skin support, so you should swap them, and indicate that ``judgement-overlay`` is used as a fallback if no judgement specific image is used, and it's meant for backward comptability. This can also be confusing since ``judgement-overlay-background`` force a per-judge image.

Reply:
> 1. Change `Background image` to `Container image`, as saying "background" for an overlay image is confusing.

In this case it's fine due to the filenaming scheme, however it is still a background image for the numbers which is the logic the rest of the wiki goes by. Ideally it would be a wiki wide change.
> 2. The Overlay is not really `Automatically colored accordingt o the JudgeColors`, but the animation is, the counter INACTIVE color is half the JudgeColor, and then the ACTIVE color, which is when an hitbobject is hit, is the full JudgeColor (This only apply if `JudgementCounterFadeToAlpha` is set to false)

Both inactive and active color is still determined by skin.ini JudgeColors.
> 3. Technically this is supposed to be the default behavior now, `judgement-overlay` is only there for default/old skin support, so you should swap them, and indicate that `judgement-overlay` is used as a fallback if no judgement specific image is used, and it's meant for backward comptability. This can also be confusing since `judgement-overlay-background` force a per-judge image.

Don't have to indicate fallback, just the behavior.

```suggestion
- Container image that displays the current judgements the player has.
- Can be skinned per judgement: `/Judgements/judgement-overlay-{marv, perf, great, good, okay, miss}.png`
- Colored according to the [JudgeColors skin.ini value]() when not using per judgement images.
```